tools/contracts/benchmarks/jmpfest-gen.ss

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env gxi
```

Comment:
I don't know how to maintain this, and I don't have gxi installed to run this. Can you please port to Rust -- the language of this repo?

Reply:
no to porting, i am not going to waste time on this. 
Its job is done, it generated the contracts; it is here as a reference and in case we need it in the future.

Reply:
Let's add a comment explaining that this was used to generate the jumpfest contract.

Reply:
ok.

Reply:
done.